build.gradle

```diff
@@ -519,10 +519,11 @@ project('standalone') {
     mainClassName = "io.pravega.local.LocalPravegaEmulator"
     applicationDefaultJvmArgs = ["-server", "-Xmx4g", "-XX:+HeapDumpOnOutOfMemoryError",
                                  "-Dlogback.configurationFile=PRAVEGA_APP_HOME/conf/logback.xml",
+                                 "-Dsinglenode.configurationFile=PRAVEGA_APP_HOME/conf/standalone-config.properties",
                                  "-Dlog.dir=PRAVEGA_APP_HOME/logs",
                                  "-Dlog.name=pravega",
                                  "-Xdebug",
-                                 "-Xrunjdwp:server=y,transport=dt_socket,address=8000,suspend=y"]
+                                 "-Xrunjdwp:server=y,transport=dt_socket,address=8000,suspend=n"]
```

Comment:
This change has been already merged here:

https://github.com/pravega/pravega/pull/2489

Reply:
Fixed. Synced with the latest master.